guides/common/modules/proc_preparing-discovery-image-for-pxe-boot.adoc

```diff
@@ -0,0 +1,33 @@
+[id="preparing-discovery-image-for-pxe-boot_{context}"]
+= Preparing Discovery image for PXE boot
```

Comment:
Do you really need a separate procedure module for creating a directory, copying files, and editing a config file? I recommend that you combine this with another procedure if possible.

Reply:
It depends on how orcharhiho handles it. For now, I'm keeping it until further notice from ATIX.

Reply:
If orcharhino needs the extra steps, we can use 'ifdefs'. I doubt that orcharhino would need an extra module.